README.md

```diff
@@ -66,7 +66,7 @@ const commands = [
 	},
 ];
 
-const rest = new REST({ version: '10' }).setToken('token');
+const rest = new REST({ version: '10' }).setToken(TOKEN);
```

Comment:
```suggestion
const rest = new REST({ version: '10' }).setToken('TOKEN');
```

Reply:
hmm idk

Reply:
I think it would be good to use it as one expression. This confuses beginners.